sklearn/cluster/_kmeans.py

```diff
@@ -698,6 +778,10 @@ class KMeans(TransformerMixin, ClusterMixin, BaseEstimator):
 
         False : never precompute distances.
 
+        .. deprecated:: 0.23
+            'precompute_distances' was deprecated in version 0.22 and will be
+            removed in 0.25.
```

Comment:
It currently has no effect, unless I'm mistaken, which should also be indicated.